src/test/datascience/notebook/executionService.vscode.test.ts

```diff
@@ -265,7 +265,7 @@ suite('DataScience - VSCode Notebook - (Execution) (slow)', function () {
         );
 
         // Interrupt the kernel).
-        await commands.executeCommand('notebook.cancelExecution');
+        await commands.executeCommand('jupyter.notebookeditor.interruptkernel');
```

Comment:
The VSC command doesn't work, i guess its an internal command and not meant to be invokved.
Even while debugging it doesn't always work (used our command that is invoked from the Command Palette.